README.md

```diff
@@ -50,14 +50,22 @@ The Sir Syed Global Scholar Award (SSGSA) is annually administered by the Sir Sy
      - Classname bg-green-860 mention color "#00a200"
   
   ### Contribution
-   - Create a branch with the specific name (according to the changes)
+   - First fork the repository and setup it if you have not done yet, otherwise make sure that your forked repo is up to date with this repo.
+   - Next find an issue that you are interested in addressing or a feature that you would like to add.
+   - Open that issue on github if not already open.
+   - Create a branch with the specific name (according to the changes) for your fix
```

Comment:
Try to follow this blog before naming the git branch.
https://deepsource.io/blog/git-branch-naming-conventions